doc/manual/rl-next/nix-config-show.md

```diff
@@ -0,0 +1,8 @@
+synopsis: `nix config show`
+issues: #7672
+prs: #9477
+description: {
+
+`nix show-config` was renamed to `nix config show` to be more consistent with the rest of the command-line interface.
```

Comment:
We should also mention the renaming of `nix doctor` here.

Reply:
```suggestion
`nix show-config` was renamed to `nix config show`, and `nix doctor` was renamed to `nix config check`, to be more consistent with the rest of the command-line interface.
```